pages/developers/indexer/_meta.json

```diff
@@ -0,0 +1,4 @@
+{
+    "indexer_api": "Indexer API",
```

Comment:
Still plenty of reference to testnet API endpoint in this doc. Filed ticket to add a endpoint in testnet specific folder and link to that endpoint in this doc